user/templates/windows/qubes-windows-tools-4-1.md

```diff
@@ -0,0 +1,341 @@
+To facilitate such separation of user profiles, Qubes Windows Tools provide an option to automatically move `C:\Users` directory to the 2nd disk backed by `private.img`. 
+It's a selectable feature of the installer. For Windows 7, it requires the private disk to be renamed to `Q:` before QWT installation (see above); for Windows 8.1, 10 and 11, this renaming occurs during QWT installation automatically.
+If that feature is selected during installation, completion of the process requires two reboots:
+
+-   The private disk is initialized and formatted on the first reboot after tools installation. It can't be done **during** the installation because Xen mass storage drivers are not yet active.
+-   User profiles are moved to the private disk on the next reboot after the private disk is initialized. 
+Reboot is required because the "mover utility" runs very early in the boot process so OS can't yet lock any files in there. 
+This can take some time depending on the profiles' size and because the GUI agent is not yet active dom0/Qubes Manager may complain that the AppVM failed to boot. 
+That's a false alarm (you can increase the AppVM's default boot timeout using `qvm-prefs`), the VM should appear "green" in Qubes Manager shortly after.
+
+It also makes sense to disable Automatic Updates for all the template-based AppVMs -- of course this should be done in the Template VM, not in individual AppVMs, because the system-wide settings are stored in the root filesystem (which holds the system-wide registry hives). Then, periodically check for updates in the Template VM and the changes will be carried over to any child AppVMs.
+
+Once the template has been created and installed it is easy to create AppVMs based on it, by selecting the type "AppVM" and a suitable template.
+
+## Using Windows disposables
+
+Windows qubes can be used as disposables, like any other Linux-based qubes. On creating a template for Windows disposables, certain preparations have to be executed:
+
+- Create an AppVM based on a Windows TemplateVM.
+- Start this AppVM and insert a link to the command prompt executable in the `Autostart` directory of the Windows menu tree:
+	- **For Windows 7:**
+	 	- If the Windows qube started in seamless mode, hit the Windows keyboard key while the cursor is positioned in a window of this VM. In non-seamless mode, klick on the Start button. In both cases, the Windows menu will be displayed.
+		- Position into the `Autostart` submenu.
+	- **For Windows 8.1, 10 or 11:**
+	 	- Type Win+R to open the execution Prompt.
+		- Type `shell:startup`.
+		- An explorer window will open, which is positioned to the `Autostart` folder.
+	- Right-click and select the option "New -> Link".
+	- Select `C:\Windows\System32\CMD.exe`as executable.
+	- Name the link, e.g. as `Command Prompt`.
+	- Close the Window with `OK`.
+	- Shut down this AppVM.
```

Comment:
This CMD window is necessary for dispVMs, because, without it, the dispVM will close before starting the intended application. The sequence described here must be executed in the AppVM used as a template for the dispVM so that the CMD window is created in each dispVM based on it. I admit that this is ugly, but so far I have not found any way around it.